### PR TITLE
Added feature of set reply-to on the message

### PR DESCRIPTION
**Pull request by nielsonsantana**
I have created the feature to set the reply-to on the message.
Also, I created a field named reply_to on the model MailerMessage, and the migration.

### CODE REVIEW

**Comment by dstegelman** [comment]
Hi @nielsonsantana thanks for your contribution!

Could you add a test to verify the header and then a migration for Django 1.7 and up?

**Comment by nielsonsantana** [comment]
Hi @dstegelman.
I did the changes.

File: mailqueue/tests/test_mail.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+from mailqueue.models import MailerMessage
+from .utils import create_email
+
+
+class MailerMessageTestCase(TestCase):
+    
+    def setUp(self):
```
Comment (dstegelman) [inline mailqueue/tests/test_mail.py]: This method isn't necessary.  It can be removed.

Comment (nielsonsantana) [inline mailqueue/tests/test_mail.py]: Method removed.

File: mailqueue/models.py [hunk 2]
```diff
@@ -99,7 +100,8 @@ def _send(self):
 
             subject, from_email = self.subject, self.from_address
             text_content = self.content
-            msg = EmailMultiAlternatives(subject, text_content, from_email)
+            reply_to = self.reply_to
+            msg = EmailMultiAlternatives(subject, text_content, from_email, headers={"reply-to": reply_to})
```
Comment (dstegelman) [inline mailqueue/models.py]: Sorry I didn't catch this before, but you'll want to add a conditional here to that blank ```reply-to``` headers aren't sent.

Something like...
```python
msg = EmailMultiAlternatives(subject, text_content, from_email)
if self.reply_to:
    msg.headers = {'reply-to': self.reply_to}
```

Comment (nielsonsantana) [inline mailqueue/models.py]: Check field reply_to added.

**Pull request merged**